lib/lib-schema-id/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.sagebionetworks</groupId>
+		<artifactId>lib</artifactId>
+		<version>develop-SNAPSHOT</version>
+	</parent>
+	<artifactId>lib-schema-id</artifactId>
+	<name>lib-schema-id</name>
+	<url>http://maven.apache.org</url>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<jacoco.branch.minumum>0.50</jacoco.branch.minumum>
+		<jacoco.line.minumum>0.50</jacoco.line.minumum>
```

Comment:
You could potentially exclude some paths in the coverage, e.g. stuff autogenerated that does not need to be tested by us